cinema/models.py

```diff
@@ -84,23 +84,45 @@
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
-        for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
-            (self.row, "row", "rows"),
-            (self.seat, "seat", "seats_in_row"),
-        ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
+    @staticmethod
+    def validate_data(seat: int, seats_in_row: int, rows: int, row: int,  error_to_raise ):
+        if not (1 <= seat <= seats_in_row):
+            raise error_to_raise(
+                {
+                    seat: f"seat must be in range: [1, {seats_in_row}], not {seat}"
+                }
+            )
+        if not (1 <= row <= rows):
+            raise error_to_raise(
+                {
+                    row: f"row must be in range: [1, {rows}], not {row}"
+                }
             )
-            if not (1 <= ticket_attr_value <= count_attrs):
-                raise ValidationError(
-                    {
-                        ticket_attr_name: f"{ticket_attr_name} "
-                        f"number must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
-                    }
-                )
+
+    def clean(self):
+        Ticket.validate_data(
+            self.seat,
+            self.row,
+            self.movie_session.cinema_hall.seats_in_row,
+            self.movie_session.cinema_hall.rows,
+            ValueError
```

Comment:
Instead of raising `ValueError`, you should raise `ValidationError` to properly integrate with Django's validation system.